### DISCUSSION TITLE
Translation flags across components

### DISCUSSION THREAD

**Discussion by rffontenelle** [WeblateOrg/weblate#12269] [Q&A] [answered]
Are translation flags (e.g. custom placeholders) shared across a project's components when their "Source code repository" field (from Settings > Version Control) is set with a [internal URL](https://docs.weblate.org/en/weblate-5.6.2/vcs.html#weblate-internal-urls)?

I would like to centralize the translation flags settings in the main project component, but I'm not sure how to get it applied to other components as well.

**Comment by nijel** [comment, marked as answer]
No, these are not shared. But this might be an useful addition to https://github.com/WeblateOrg/weblate/issues/10897
